src/Components/Search_bar/Search_bar.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import './search_bar.css';
+
+const Searchbar = ({ handleSearchEntry, searchByWord, searchField }) => {
+    return (
+        <section className="top-container">
+            <input className="search-bar"
+                type={"text"}
+                placeholder={"SEARCH TITLES"}
+                name={"searchField"}
+                value={searchField}
+                onChange={event => handleSearchEntry(event)}
+            ></input>
+            <button className="submit-btn" onClick={event => searchByWord(event)}>Submit</button>
+        </section>
```

Comment:
if you have a input - you almost always need a class based controlled form to track it's data - this could be a good refactor